Tools/CISupport/ews-build/steps.py

```diff
@@ -1383,12 +1385,25 @@ def __init__(self, skipBuildIfNoResult=True):
 
     def find_test_names_from_patch(self, patch):
         tests = []
+        test_expectation_changes = False
         for line in patch.splitlines():
+            # Check if changes are in a TestExpectations file
+            file_change_line = re.search(self.RE_FILE_CHANGE, line, re.IGNORECASE)
```

Comment:
Have we double checked that the "patch diff" for a GitHub PR actually has the diff of each file? I seem to recall we only implemented which files were added/removed/modified, not what that modification actually was

Reply:
Addressed IRL. For posterity, this is true and the implementation has been changed to look at the results of `run-webkit-tests --print-expectations`.